mediorum/server/repair.go

```diff
@@ -345,52 +337,6 @@ func (ss *MediorumServer) repairCid(cid string, placementHosts []string, tracker
 		}
 	}
 
-	// replicate under-replicated blobs:
```

Comment:
why remove this?

Reply:
In practice it causes nodes to spam each other too much in prod...
Ideally repair.go should only make network requests to actually get a file.

I commented this out a while ago and it's just been sitting here commented out for a while.